plotter.py

```diff
@@ -0,0 +1,29 @@
+"""
+Plotter functions.
+"""
+
+import pandas as pd
+import plotly
+import plotly.express as px
+
+def line(raw_data, tittle='2D-line',
+         x_axial='X', y_axial='Y', instruments='instruments'):
+    '''
+    Function for 2D-line.
+
+    Raw_data is a dict of tuple list.
+    The keys means names of data, and each tuple means a coordinate point
+    '''
```

Comment:
We have many args now, so it may be better to describe inputs in a better format.
Can you follow [this guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)?